lms/djangoapps/instructor/access.py

```diff
@@ -80,7 +81,9 @@ def _change_access(course, user, level, action, send_email=True):
 
     if action == 'allow':
         if level == 'ccx_coach':
-            email_params = get_email_params(course, True)
+            ccx_course_object = CustomCourseForEdX.objects.get(pk=course.id.ccx)
```

Comment:
is it possible that the course with given id doesn't exist? If so, is it handled somehow?

Reply:
good catch!, 
yes it possible, handler added.